itl/plugins-contrib.d/web.conf

```diff
@@ -396,13 +396,9 @@ object CheckCommand "ssl_cert" {
 			value = "$ssl_cert_critical$"
 			description = "Minimum number of days a certificate has to be valid to issue a critical status"
 		}
-		"-n" = {
+		"--match" = {
 			value = "$ssl_cert_cn$"
-			description = "Pattern to match the CN of the certificate"
-		}
-		"--altnames" = {
-			set_if = "$ssl_cert_altnames$"
-			description = "Matches the pattern specified in -n with alternate"
```

Comment:
What about passing --match here as well?

Reply:
Please consider this one:

* https://github.com/liip-forks/icinga2/pull/1

You can merge it via rebase-and-merge not to produce an extra merge commit.